modules/ROOT/pages/vpn-about.adoc

```diff
@@ -16,31 +22,89 @@ To create and maintain a secure connection with your external network, the Anypo
 
 [NOTE]
 Network Address Translation (NAT) is not supported.
+////
+
+== Anypoint VPN Features
 
-== Supported IPsec Configurations
+=== IPsec Settings
 
-Anypoint VPN supports the IPsec configurations for encrypting data communications between Anypoint Platform and your network:
+Anypoint VPN supports any combination of the following IPsec settings:
 
 * IKE version 1
-* AES 128 and 256-bit encryption
-* SHA and SHA-2 hashing
-* Diffie-Hellman (DH) groups:
-** Phase1 DH groups 2, 14-18, 22, 23, 24
-** Phase2 DH groups 2, 5, 14-18, 22, 23, 24
+* IKE version 2 for route-based VPNs only
+* AES 128 or 256-bit encryption
+* SHA or SHA-2 (256) hashing
+* Diffie-Hellman Phase1 groups 2, 14-18, 22, 23, 24
+* Diffie-Hellman Phase2 groups 2, 5, 14-18, 22, 23, 24
+
+[NOTE]
+Your endpoint must initiate traffic to establish and maintain the connection so the Anypoint VPN endpoint can act as a responder.
+
+=== Anypoint VPN Routing
 
+Anypoint VPN supports dynamic or static routing for VPN connections.
+
+* *Dynamic routing* Your device uses Border Gateway Protocol (BGP) to advertise routes to Anypoint VPN. We recommend using BGP routing if your device supports this protocol.
+* *Static routing* - Requires you to specify the routes (subnets) in your network that are accessible through Anypoint VPN. +
 [NOTE]
-Because the Anypoint VPN endpoint is the responder, be sure to establish and maintain connections by initiating traffic from a remote device.
+A maximum of 95 static routes are permitted per VPC, regardless of the number of VPN connections. We recommend consolidating networks to the fewest number possible.
```

Comment:
Instead of "We recommend" why not "Consolidate networks to the fewest number possible if you would otherwise exceed the limit." or similar.